test/test_flake8.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from pathlib import Path
+import sys
+
+from flake8 import LOG
+from flake8.api.legacy import get_style_guide
+
+
+# Avoid debug and info messages from flake8 internals
+LOG.setLevel(logging.WARN)
+
+
+def test_flake8():
+    style_guide = get_style_guide(
+        ignore=['D100', 'D104'],
```

Comment:
I recommend to use `extend_ignore` to use the default ignore list provided by `flake8`.

Same below.